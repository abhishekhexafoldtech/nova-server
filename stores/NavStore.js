import { defineStore } from "pinia";
import bootstrap from "bootstrap/dist/js/bootstrap.min.js";

export const useNavStore = defineStore("NavStore", {
  // state
  state: () => {
    return {
      bootstrap: bootstrap,
      isSidenavCollapsed: false,
      showConfig: false,
      sidenavType: 'bg-white',
      navFixed: false,
      darkMode: false,
      miniNotification: false,
      breadCrumb: "Nova-Gas"
    };
  },

  actions: {
    toggleSidenavCollapsing() {
      this.isSidenavCollapsed = !this.isSidenavCollapsed;
    },
    toggleShowConfig() {
      this.showConfig = !this.showConfig;
    },
    toggleSidebarColor(payload) {
      this.sidebarColor = payload;
    },
    setSidenavType(payload) {
      this.sidenavType = payload;
    },
    toggleDarkMode() {
      this.darkMode = !this.darkMode;
    },
    toggleNavFixed() {
      this.navFixed = !this.navFixed;
    },
    setBreadCrumbs(data) {
      this.breadCrumb = data
    }
  },
});

