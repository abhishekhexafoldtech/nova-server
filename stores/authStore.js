import { defineStore } from 'pinia';
import { getToken, removeToken, setToken } from '@/utils/auth'
import {  getUserInfo, loginByWeb, logoutByWeb } from '@/api/login'

export const useAuthStore = defineStore("auth", {

  state: () => ({
    token: getToken(),
    user_info: {}, // complete user details
    page_refreshing: true,
  }),

  getters: {
    getLoginToken() {
      return this.token
    },
  },

  actions: {
	// Login
	Login(userCredInfo) {
		return new Promise((resolve, reject) => {
			// make login data set
			let data = {
				email: userCredInfo.email,
				phone: userCredInfo.phone,
				password: userCredInfo.password,
			}
			// Lets fire login with info
			loginByWeb(data).then(response => {
				if(response.status == 200) {
					let responseData = response.data
					// console.log("responseData", responseData)
					let go = true
						if (go) {
							// SITE TOKEN
							this.token = responseData.access_token
							this.user_info = responseData.data
							setToken(responseData.access_token)

							const person = { 
									"id": responseData.data.id,
									"first_name": responseData.data.first_name,
									"last_name": responseData.data.last_name,
									"email": responseData.data.email,
									"phone": responseData.data.phone,
								}
							
							// console.log('person', person)
							localStorage.setItem("user", JSON.stringify(person));

							resolve()
						}
					}
						else {
						alert('Something went wrong!')
						reject()
					}
			}).catch(error => {
				reject(error)
			})
		})
	},

	// Logout
	LogOut() {
		return new Promise((resolve, reject) => {
			this.token = ""
			removeToken()
			localStorage.clear();
			resolve()
			// logoutByWeb().then((response) => {
			// 	this.token = ""
			// 	removeToken()
			// 	resolve(response)
			// }).catch(error => {
			// 	console.log(error)
			// 	reject(error)
			// })
			// .finally(() => {
			// })
		})
	},

	// Navbar loaded or refreshed : both cases fired
	pageLoadedORRefreshed() {
		// Get user info
		return new Promise((resolve) => {
			this.page_refreshing = true;
			// Get user info
			// getUserInfo().then((response) => {
			// this.user_info = response.data.data
			// 	resolve(response);
			// })
			// Get user info from local storage
			let temp_data = localStorage.getItem("user")
			this.user_info = JSON.parse(temp_data)
		});
	},
  },
});
