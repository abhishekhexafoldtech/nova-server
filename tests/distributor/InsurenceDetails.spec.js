import { shallowMount } from "@vue/test-utils";
import { describe, it, expect } from "vitest";
import { pinia } from "@/tests/mockpinia";
import { useNavStore } from "@/stores/NavStore";
import InsuranceDetails from "@/pages/distributor/add-distributor/insurence-details.vue";

describe("InsuranceDetails Component", () => {
  const navStore = useNavStore(pinia);
  it("renders all input fields", () => {
    const wrapper = shallowMount(InsuranceDetails);
    //  InsuranceDetails component uses Element Plus UI elements, you can use specific queries to check for the presence of those elements.
    //  Check if an Element Plus input element with a label name is present
    expect(wrapper.find('[placeholder="Name of insurance company"]').exists()).toBe(true);
    expect(wrapper.find('[placeholder="Policy number"]').exists()).toBe(true);
    expect(wrapper.find('[placeholder="Expiry of insurance"]').exists()).toBe(true );

    // Assert that the rendered HTML matches a file snapshot if you're using snapshot testing.
    expect(wrapper.html()).toMatchSnapshot();
  });

  it("renders the SingleFileUpload Component", () => {
    const wrapper = shallowMount(InsuranceDetails);
    // Find the <SingleFileUpload> component within the parent component
    const singleFileUpload = wrapper.findComponent({
      name: "SingleFileUpload",
    });

    // Assert that the SingleFileUpload component exists within the parent component
    expect(singleFileUpload.exists()).toBe(true);
    expect(wrapper.html()).toMatchSnapshot();
  });

  it("renders insurance card component", () => {
    const wrapper = shallowMount(InsuranceDetails);

    // Find the SingleFileUpload component based on its id
    const singleFileUpload = wrapper.find("#insurance_card");

    // Use assertions to verify that the SingleFileUpload component exists
    expect(singleFileUpload.exists()).toBe(true);
    expect(wrapper.html()).toMatchSnapshot();
  });

  //check save button present or not
  it("renders the Save button", async () => {
    const wrapper = shallowMount(InsuranceDetails);

    // Check if the "Continue" button is present
    expect(wrapper.find(".btn-primary").text()).toBe("Save");
    await wrapper.find(".btn-primary").trigger("click");
    expect(wrapper.html()).toMatchSnapshot();
  });
});
