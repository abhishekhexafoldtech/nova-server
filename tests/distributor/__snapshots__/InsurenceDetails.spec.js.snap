// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`InsuranceDetails Component > renders all input fields 1`] = `
"<section data-v-3c2eb23c=\\"\\" class=\\"edit_mang_wrap\\">
  <div data-v-3c2eb23c=\\"\\" class=\\"mang_inner comp_inner\\">
    <div data-v-3c2eb23c=\\"\\" class=\\"mang_title\\">
      <h3 data-v-3c2eb23c=\\"\\" class=\\"mb-3\\">Insurance details</h3>
      <p data-v-3c2eb23c=\\"\\">Enter your business insurance details</p>
    </div>
    <div data-v-3c2eb23c=\\"\\" class=\\"w854\\">
      <el-form data-v-3c2eb23c=\\"\\" label-position=\\"top\\" label-width=\\"100px\\" model=\\"[object Object]\\" rules=\\"[object Object]\\">
        <el-row data-v-3c2eb23c=\\"\\" gutter=\\"50\\">
          <!--Name of Insurance Company  -->
          <el-col data-v-3c2eb23c=\\"\\" xs=\\"24\\" sm=\\"24\\" md=\\"12\\" lg=\\"12\\" xl=\\"12\\">
            <div data-v-3c2eb23c=\\"\\" class=\\"fieldrow\\">
              <el-form-item data-v-3c2eb23c=\\"\\" label=\\"Name of Insurance Company\\" prop=\\"name_of_insurence_company\\">
                <el-input data-v-3c2eb23c=\\"\\" modelvalue=\\"\\" placeholder=\\"Name of insurance company\\"></el-input>
              </el-form-item>
            </div>
          </el-col><!-- Policy Number -->
          <el-col data-v-3c2eb23c=\\"\\" xs=\\"24\\" sm=\\"24\\" md=\\"12\\" lg=\\"12\\" xl=\\"12\\">
            <div data-v-3c2eb23c=\\"\\" class=\\"fieldrow\\">
              <el-form-item data-v-3c2eb23c=\\"\\" label=\\"Policy Number\\" prop=\\"policy_number\\">
                <el-input data-v-3c2eb23c=\\"\\" modelvalue=\\"\\" placeholder=\\"Policy number\\"></el-input>
              </el-form-item>
            </div>
          </el-col><!-- insurance card -->
          <el-col data-v-3c2eb23c=\\"\\" xs=\\"24\\" sm=\\"24\\" md=\\"12\\" lg=\\"12\\" xl=\\"12\\">
            <div data-v-3c2eb23c=\\"\\" class=\\"comp_form_box phase3_upload\\">
              <div data-v-3c2eb23c=\\"\\" class=\\"fieldrow\\">
                <p data-v-3c2eb23c=\\"\\" class=\\"frm_label\\">Insurance Card</p>
                <single-file-upload-stub data-v-3c2eb23c=\\"\\" iconclass=\\"iconClass\\" progress=\\"true\\" id=\\"insurance_card\\" value=\\"[object Object]\\"></single-file-upload-stub>
              </div>
              <!--v-if-->
            </div>
          </el-col><!-- Expiry of insurance -->
          <el-col data-v-3c2eb23c=\\"\\" xs=\\"24\\" sm=\\"24\\" md=\\"12\\" lg=\\"12\\" xl=\\"12\\">
            <div data-v-3c2eb23c=\\"\\" class=\\"fieldrow\\">
              <!-- <el-form-item label=\\"Expiry of Insurance\\" prop=\\"expiry_of_insurence\\">
                  <el-input type=\\"date\\" v-model=\\"formData.expiry_of_insurence\\" placeholder=\\"Expiry of insurance\\" />
                </el-form-item> -->
              <el-form-item data-v-3c2eb23c=\\"\\" label=\\"Expiry of Insurance\\" prop=\\"expiry_of_insurence\\">
                <el-date-picker data-v-3c2eb23c=\\"\\" value-format=\\"YYYY-MM-DD\\" modelvalue=\\"\\" type=\\"date\\" placeholder=\\"Expiry of insurance\\" size=\\"default\\"></el-date-picker>
              </el-form-item>
            </div>
          </el-col>
        </el-row>
      </el-form>
      <div data-v-3c2eb23c=\\"\\" class=\\"comp_footer\\">
        <nuxt-link data-v-3c2eb23c=\\"\\" class=\\"btn btn-default\\" to=\\"/distributor\\">Cancel</nuxt-link>
        <el-button data-v-3c2eb23c=\\"\\" class=\\"btn btn-primary\\">Save</el-button>
      </div>
    </div>
  </div>
</section>
<dialog-status-stub data-v-3c2eb23c=\\"\\" dialogtitle=\\"Have a depot?\\" dialogimage=\\"/assets/svg/insurence.svg\\" dialogtext=\\"Do you want to assign zone or
 do you want to onboard depot\\" leftbuttonname=\\"Assign depot\\" rightbuttonname=\\"Onboard depot\\" dialogvisible=\\"false\\" dialogtextsize=\\"17\\" dialogtitlesize=\\"22\\"></dialog-status-stub>"
`;

exports[`InsuranceDetails Component > renders insurance card component 1`] = `
"<section data-v-3c2eb23c=\\"\\" class=\\"edit_mang_wrap\\">
  <div data-v-3c2eb23c=\\"\\" class=\\"mang_inner comp_inner\\">
    <div data-v-3c2eb23c=\\"\\" class=\\"mang_title\\">
      <h3 data-v-3c2eb23c=\\"\\" class=\\"mb-3\\">Insurance details</h3>
      <p data-v-3c2eb23c=\\"\\">Enter your business insurance details</p>
    </div>
    <div data-v-3c2eb23c=\\"\\" class=\\"w854\\">
      <el-form data-v-3c2eb23c=\\"\\" label-position=\\"top\\" label-width=\\"100px\\" model=\\"[object Object]\\" rules=\\"[object Object]\\">
        <el-row data-v-3c2eb23c=\\"\\" gutter=\\"50\\">
          <!--Name of Insurance Company  -->
          <el-col data-v-3c2eb23c=\\"\\" xs=\\"24\\" sm=\\"24\\" md=\\"12\\" lg=\\"12\\" xl=\\"12\\">
            <div data-v-3c2eb23c=\\"\\" class=\\"fieldrow\\">
              <el-form-item data-v-3c2eb23c=\\"\\" label=\\"Name of Insurance Company\\" prop=\\"name_of_insurence_company\\">
                <el-input data-v-3c2eb23c=\\"\\" modelvalue=\\"\\" placeholder=\\"Name of insurance company\\"></el-input>
              </el-form-item>
            </div>
          </el-col><!-- Policy Number -->
          <el-col data-v-3c2eb23c=\\"\\" xs=\\"24\\" sm=\\"24\\" md=\\"12\\" lg=\\"12\\" xl=\\"12\\">
            <div data-v-3c2eb23c=\\"\\" class=\\"fieldrow\\">
              <el-form-item data-v-3c2eb23c=\\"\\" label=\\"Policy Number\\" prop=\\"policy_number\\">
                <el-input data-v-3c2eb23c=\\"\\" modelvalue=\\"\\" placeholder=\\"Policy number\\"></el-input>
              </el-form-item>
            </div>
          </el-col><!-- insurance card -->
          <el-col data-v-3c2eb23c=\\"\\" xs=\\"24\\" sm=\\"24\\" md=\\"12\\" lg=\\"12\\" xl=\\"12\\">
            <div data-v-3c2eb23c=\\"\\" class=\\"comp_form_box phase3_upload\\">
              <div data-v-3c2eb23c=\\"\\" class=\\"fieldrow\\">
                <p data-v-3c2eb23c=\\"\\" class=\\"frm_label\\">Insurance Card</p>
                <single-file-upload-stub data-v-3c2eb23c=\\"\\" iconclass=\\"iconClass\\" progress=\\"true\\" id=\\"insurance_card\\" value=\\"[object Object]\\"></single-file-upload-stub>
              </div>
              <!--v-if-->
            </div>
          </el-col><!-- Expiry of insurance -->
          <el-col data-v-3c2eb23c=\\"\\" xs=\\"24\\" sm=\\"24\\" md=\\"12\\" lg=\\"12\\" xl=\\"12\\">
            <div data-v-3c2eb23c=\\"\\" class=\\"fieldrow\\">
              <!-- <el-form-item label=\\"Expiry of Insurance\\" prop=\\"expiry_of_insurence\\">
                  <el-input type=\\"date\\" v-model=\\"formData.expiry_of_insurence\\" placeholder=\\"Expiry of insurance\\" />
                </el-form-item> -->
              <el-form-item data-v-3c2eb23c=\\"\\" label=\\"Expiry of Insurance\\" prop=\\"expiry_of_insurence\\">
                <el-date-picker data-v-3c2eb23c=\\"\\" value-format=\\"YYYY-MM-DD\\" modelvalue=\\"\\" type=\\"date\\" placeholder=\\"Expiry of insurance\\" size=\\"default\\"></el-date-picker>
              </el-form-item>
            </div>
          </el-col>
        </el-row>
      </el-form>
      <div data-v-3c2eb23c=\\"\\" class=\\"comp_footer\\">
        <nuxt-link data-v-3c2eb23c=\\"\\" class=\\"btn btn-default\\" to=\\"/distributor\\">Cancel</nuxt-link>
        <el-button data-v-3c2eb23c=\\"\\" class=\\"btn btn-primary\\">Save</el-button>
      </div>
    </div>
  </div>
</section>
<dialog-status-stub data-v-3c2eb23c=\\"\\" dialogtitle=\\"Have a depot?\\" dialogimage=\\"/assets/svg/insurence.svg\\" dialogtext=\\"Do you want to assign zone or
 do you want to onboard depot\\" leftbuttonname=\\"Assign depot\\" rightbuttonname=\\"Onboard depot\\" dialogvisible=\\"false\\" dialogtextsize=\\"17\\" dialogtitlesize=\\"22\\"></dialog-status-stub>"
`;

exports[`InsuranceDetails Component > renders the Save button 1`] = `
"<section data-v-3c2eb23c=\\"\\" class=\\"edit_mang_wrap\\">
  <div data-v-3c2eb23c=\\"\\" class=\\"mang_inner comp_inner\\">
    <div data-v-3c2eb23c=\\"\\" class=\\"mang_title\\">
      <h3 data-v-3c2eb23c=\\"\\" class=\\"mb-3\\">Insurance details</h3>
      <p data-v-3c2eb23c=\\"\\">Enter your business insurance details</p>
    </div>
    <div data-v-3c2eb23c=\\"\\" class=\\"w854\\">
      <el-form data-v-3c2eb23c=\\"\\" label-position=\\"top\\" label-width=\\"100px\\" model=\\"[object Object]\\" rules=\\"[object Object]\\">
        <el-row data-v-3c2eb23c=\\"\\" gutter=\\"50\\">
          <!--Name of Insurance Company  -->
          <el-col data-v-3c2eb23c=\\"\\" xs=\\"24\\" sm=\\"24\\" md=\\"12\\" lg=\\"12\\" xl=\\"12\\">
            <div data-v-3c2eb23c=\\"\\" class=\\"fieldrow\\">
              <el-form-item data-v-3c2eb23c=\\"\\" label=\\"Name of Insurance Company\\" prop=\\"name_of_insurence_company\\">
                <el-input data-v-3c2eb23c=\\"\\" modelvalue=\\"\\" placeholder=\\"Name of insurance company\\"></el-input>
              </el-form-item>
            </div>
          </el-col><!-- Policy Number -->
          <el-col data-v-3c2eb23c=\\"\\" xs=\\"24\\" sm=\\"24\\" md=\\"12\\" lg=\\"12\\" xl=\\"12\\">
            <div data-v-3c2eb23c=\\"\\" class=\\"fieldrow\\">
              <el-form-item data-v-3c2eb23c=\\"\\" label=\\"Policy Number\\" prop=\\"policy_number\\">
                <el-input data-v-3c2eb23c=\\"\\" modelvalue=\\"\\" placeholder=\\"Policy number\\"></el-input>
              </el-form-item>
            </div>
          </el-col><!-- insurance card -->
          <el-col data-v-3c2eb23c=\\"\\" xs=\\"24\\" sm=\\"24\\" md=\\"12\\" lg=\\"12\\" xl=\\"12\\">
            <div data-v-3c2eb23c=\\"\\" class=\\"comp_form_box phase3_upload\\">
              <div data-v-3c2eb23c=\\"\\" class=\\"fieldrow\\">
                <p data-v-3c2eb23c=\\"\\" class=\\"frm_label\\">Insurance Card</p>
                <single-file-upload-stub data-v-3c2eb23c=\\"\\" iconclass=\\"iconClass\\" progress=\\"true\\" id=\\"insurance_card\\" value=\\"[object Object]\\"></single-file-upload-stub>
              </div>
              <!--v-if-->
            </div>
          </el-col><!-- Expiry of insurance -->
          <el-col data-v-3c2eb23c=\\"\\" xs=\\"24\\" sm=\\"24\\" md=\\"12\\" lg=\\"12\\" xl=\\"12\\">
            <div data-v-3c2eb23c=\\"\\" class=\\"fieldrow\\">
              <!-- <el-form-item label=\\"Expiry of Insurance\\" prop=\\"expiry_of_insurence\\">
                  <el-input type=\\"date\\" v-model=\\"formData.expiry_of_insurence\\" placeholder=\\"Expiry of insurance\\" />
                </el-form-item> -->
              <el-form-item data-v-3c2eb23c=\\"\\" label=\\"Expiry of Insurance\\" prop=\\"expiry_of_insurence\\">
                <el-date-picker data-v-3c2eb23c=\\"\\" value-format=\\"YYYY-MM-DD\\" modelvalue=\\"\\" type=\\"date\\" placeholder=\\"Expiry of insurance\\" size=\\"default\\"></el-date-picker>
              </el-form-item>
            </div>
          </el-col>
        </el-row>
      </el-form>
      <div data-v-3c2eb23c=\\"\\" class=\\"comp_footer\\">
        <nuxt-link data-v-3c2eb23c=\\"\\" class=\\"btn btn-default\\" to=\\"/distributor\\">Cancel</nuxt-link>
        <el-button data-v-3c2eb23c=\\"\\" class=\\"btn btn-primary\\">Save</el-button>
      </div>
    </div>
  </div>
</section>
<dialog-status-stub data-v-3c2eb23c=\\"\\" dialogtitle=\\"Have a depot?\\" dialogimage=\\"/assets/svg/insurence.svg\\" dialogtext=\\"Do you want to assign zone or
 do you want to onboard depot\\" leftbuttonname=\\"Assign depot\\" rightbuttonname=\\"Onboard depot\\" dialogvisible=\\"false\\" dialogtextsize=\\"17\\" dialogtitlesize=\\"22\\"></dialog-status-stub>"
`;

exports[`InsuranceDetails Component > renders the SingleFileUpload Component 1`] = `
"<section data-v-3c2eb23c=\\"\\" class=\\"edit_mang_wrap\\">
  <div data-v-3c2eb23c=\\"\\" class=\\"mang_inner comp_inner\\">
    <div data-v-3c2eb23c=\\"\\" class=\\"mang_title\\">
      <h3 data-v-3c2eb23c=\\"\\" class=\\"mb-3\\">Insurance details</h3>
      <p data-v-3c2eb23c=\\"\\">Enter your business insurance details</p>
    </div>
    <div data-v-3c2eb23c=\\"\\" class=\\"w854\\">
      <el-form data-v-3c2eb23c=\\"\\" label-position=\\"top\\" label-width=\\"100px\\" model=\\"[object Object]\\" rules=\\"[object Object]\\">
        <el-row data-v-3c2eb23c=\\"\\" gutter=\\"50\\">
          <!--Name of Insurance Company  -->
          <el-col data-v-3c2eb23c=\\"\\" xs=\\"24\\" sm=\\"24\\" md=\\"12\\" lg=\\"12\\" xl=\\"12\\">
            <div data-v-3c2eb23c=\\"\\" class=\\"fieldrow\\">
              <el-form-item data-v-3c2eb23c=\\"\\" label=\\"Name of Insurance Company\\" prop=\\"name_of_insurence_company\\">
                <el-input data-v-3c2eb23c=\\"\\" modelvalue=\\"\\" placeholder=\\"Name of insurance company\\"></el-input>
              </el-form-item>
            </div>
          </el-col><!-- Policy Number -->
          <el-col data-v-3c2eb23c=\\"\\" xs=\\"24\\" sm=\\"24\\" md=\\"12\\" lg=\\"12\\" xl=\\"12\\">
            <div data-v-3c2eb23c=\\"\\" class=\\"fieldrow\\">
              <el-form-item data-v-3c2eb23c=\\"\\" label=\\"Policy Number\\" prop=\\"policy_number\\">
                <el-input data-v-3c2eb23c=\\"\\" modelvalue=\\"\\" placeholder=\\"Policy number\\"></el-input>
              </el-form-item>
            </div>
          </el-col><!-- insurance card -->
          <el-col data-v-3c2eb23c=\\"\\" xs=\\"24\\" sm=\\"24\\" md=\\"12\\" lg=\\"12\\" xl=\\"12\\">
            <div data-v-3c2eb23c=\\"\\" class=\\"comp_form_box phase3_upload\\">
              <div data-v-3c2eb23c=\\"\\" class=\\"fieldrow\\">
                <p data-v-3c2eb23c=\\"\\" class=\\"frm_label\\">Insurance Card</p>
                <single-file-upload-stub data-v-3c2eb23c=\\"\\" iconclass=\\"iconClass\\" progress=\\"true\\" id=\\"insurance_card\\" value=\\"[object Object]\\"></single-file-upload-stub>
              </div>
              <!--v-if-->
            </div>
          </el-col><!-- Expiry of insurance -->
          <el-col data-v-3c2eb23c=\\"\\" xs=\\"24\\" sm=\\"24\\" md=\\"12\\" lg=\\"12\\" xl=\\"12\\">
            <div data-v-3c2eb23c=\\"\\" class=\\"fieldrow\\">
              <!-- <el-form-item label=\\"Expiry of Insurance\\" prop=\\"expiry_of_insurence\\">
                  <el-input type=\\"date\\" v-model=\\"formData.expiry_of_insurence\\" placeholder=\\"Expiry of insurance\\" />
                </el-form-item> -->
              <el-form-item data-v-3c2eb23c=\\"\\" label=\\"Expiry of Insurance\\" prop=\\"expiry_of_insurence\\">
                <el-date-picker data-v-3c2eb23c=\\"\\" value-format=\\"YYYY-MM-DD\\" modelvalue=\\"\\" type=\\"date\\" placeholder=\\"Expiry of insurance\\" size=\\"default\\"></el-date-picker>
              </el-form-item>
            </div>
          </el-col>
        </el-row>
      </el-form>
      <div data-v-3c2eb23c=\\"\\" class=\\"comp_footer\\">
        <nuxt-link data-v-3c2eb23c=\\"\\" class=\\"btn btn-default\\" to=\\"/distributor\\">Cancel</nuxt-link>
        <el-button data-v-3c2eb23c=\\"\\" class=\\"btn btn-primary\\">Save</el-button>
      </div>
    </div>
  </div>
</section>
<dialog-status-stub data-v-3c2eb23c=\\"\\" dialogtitle=\\"Have a depot?\\" dialogimage=\\"/assets/svg/insurence.svg\\" dialogtext=\\"Do you want to assign zone or
 do you want to onboard depot\\" leftbuttonname=\\"Assign depot\\" rightbuttonname=\\"Onboard depot\\" dialogvisible=\\"false\\" dialogtextsize=\\"17\\" dialogtitlesize=\\"22\\"></dialog-status-stub>"
`;
